Utilities/File.h

```diff
@@ -694,6 +701,7 @@ namespace fs
 		bool trunc(u64 length) override
 		{
 			obj.resize(length);
+			update_time(true);
```

Comment:
No, this is for POSIX allowence of deletion of files with opened file descriptors. In this case I convert the file descriptor to memory and added missing functionality so it can be used by cellFs.